src/services/tx/relaying.ts

```diff
@@ -0,0 +1,60 @@
+import { type SafeTransactionData } from '@safe-global/safe-core-sdk-types'
+import { IS_PRODUCTION, SAFE_RELAY_SERVICE_URL_PRODUCTION, SAFE_RELAY_SERVICE_URL_STAGING } from '@/config/constants'
+import { cgwDebugStorage } from '@/components/sidebar/DebugToggle'
+
+export const SAFE_RELAY_SERVICE_URL =
+  IS_PRODUCTION || cgwDebugStorage.get() ? SAFE_RELAY_SERVICE_URL_PRODUCTION : SAFE_RELAY_SERVICE_URL_STAGING
+
+// TODO: import types from relay-service
```

Comment:
Please see note in description.

Reply:
After discussion with @fmrsabino, this is going to be tackled as part of a Friday project - an auto-generated SDK.